xla/backends/profiler/gpu/device_tracer_rocm.cc

```diff
@@ -166,44 +161,46 @@ class RocmTraceCollectorImpl : public profiler::RocmTraceCollector {
             << " activity events, and aggregated them into "
             << aggregated_events_.size() << " events.";
 
+    //device ids for GPUs filled in by roctracer are not zero indexed.
+    //They are offset by number of CPUs on the machine
+    tsl::uint32 min_device_id = 2^31;
```

Comment:
This magic constant looks suspicious.
Why 29?

Reply:
Fixed